src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -494,8 +493,8 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
     uct_rc_verbs_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
     ucs_status_t status;
 
-    if (comp != NULL) {
-        return UCS_ERR_UNSUPPORTED;
+    if (!uct_rc_iface_has_tx_resources(&iface->super)) {
```

Comment:
if we have pending, it means that some resources are unavailable. Therefore, to make sure there are no pending, we must make sure all resources *are* available:
pending -> no_resources
have_resources -> no_pending

before returning OK, need to make sore there are no pending. for that, it's enough to make sure we have resources.